docs/campaign-pages.md

```diff
@@ -0,0 +1,17 @@
+# Campaign pages
+
+Collections currently renders the following campaign pages:
+
+## Transition landing page ([ /transition ](https://www.gov.uk/transition))
```

Comment:
Not your fault, but could these links be normalised? There are a few oddities in this markdown.

```suggestion
## Transition landing page ([/transition](https://www.gov.uk/transition))
```

Reply:
That's weird, and yes why not. I've amended in 39b40d4f78a7eb44228fa25272db86c6b5aca33b